frontend/package.json

```diff
@@ -9,8 +9,8 @@
     "dist-staging": "ng build --prod --env=staging --aot=false",
     "dist-prod": "ng build --prod --env=prod --aot=false",
     "docs": "yarn run typedoc --out ./src/assets/typedoc --exclude '**/*.spec.ts' ./src/ --module commonjs",
-    "e2e:ci": "ng e2e --port=4200 --watch=false --progress=false",
-    "e2e:cidkr": "ng e2e --port=4200 --watch=false --progress=false --config ./development-configurations/protractor.conf.js --specs=./e2e/authenticated/special-uses/application-noncommercial-group.e2e-spec.ts",
+    "e2e:ci": "ng e2e --port=4200",
```

Comment:
Is `--progress=false` still a valid flag? Would adding that back in remove the compiling output from CircleCI?

Reply:
Scratch that, I was getting this confused with the frontend unit test.

Reply:
Looking at the cli docs, I don't think it is. https://github.com/angular/angular-cli/wiki/e2e but i can check locally